docs/.vitepress/theme/Users.vue

```diff
@@ -0,0 +1,70 @@
+<script>
+export const users = [
+  {
+    name: "Ahrefs",
+    image: "ahrefs.svg",
```

Comment:
Can we use the newest ahrefs logo from https://ahrefs.com/logo

Reply:
Updated in https://github.com/melange-re/melange-re.github.io/pull/196/commits/e51bf5ef1fa4ce7951ee044d9090c3bf4ba865de